src/commands/githubActions/endToEndTest/testCheckers.ts

```diff
@@ -116,10 +116,42 @@ async function ensureNoErrorLogs(
 }
 
 /**
- * Test that the validators are attesting after a peri
+ * Test that the validators are attesting after 16 minutes
+ * 
  */
-async function attestanceProof(network: Network): Promise<void> {
-  if (network === "mainnet") return;
-  // TODO
+export async function attestanceProof(network: Network): Promise<void> {
+  // Check if the network is mainnet or prater
+  if (network === "mainnet") {
```

Comment:
It looks like the only thing that changes depending on the network is the URL, so there is no need to have all this code inside the `if-else-block`

Reply:
Already addressed